src/components/project/ProjectCard.tsx

```diff
@@ -35,6 +38,10 @@ const ProjectCard = ({ project, renderAsLink, showActions }: IProps): JSX.Elemen
     uuid,
   } = project;
 
+  const applicationPeriodHasStarted = new Date().getTime() > new Date(application_start_time).getTime();
+  const applicationPeriodHasEnded = new Date().getTime() > new Date(application_end_time).getTime();
```

Comment:
Here I would create a "now" variable before doing the checks here. I know that the likelihood of some issue happening here where the one millisecond change would make a difference between the start and end checks, but it is just generally good practice to use the same time to check for things that relate to eachother.

Reply:
This is now fixed.